cmd/podman/images/save.go

```diff
@@ -91,11 +97,46 @@ func save(cmd *cobra.Command, args []string) error {
 		return errors.Errorf("--compress can only be set when --format is either 'oci-dir' or 'docker-dir'")
 	}
 	if len(saveOpts.Output) == 0 {
-		fi := os.Stdout
-		if terminal.IsTerminal(int(fi.Fd())) {
+		pipeFile := "saveio"
```

Comment:
(Absolutely non-blocking: This has a single user and can be inlined — or make the file name a top-level constant.)